app/src/main/java/com/example/vlad/financemanager/data/models/User.java

```diff
@@ -4,28 +4,38 @@
 import java.util.ArrayList;
 
 public class User {
-    /** users's id */
+    /**
+     * users's id
+     */
     private int id;
-    /** user's balance */
+    /**
+     * user's balance
+     */
     private BigDecimal balance;
-    /** user's name */
+    /**
+     * user's name
+     */
     private String name;
-    /** users's accounts */
+    /**
+     * users's accounts
+     */
     private ArrayList<Account> accounts;
-    /** collection of the user's custom categories */
+    /**
+     * collection of the user's custom categories
+     */
     private ArrayList<Category> userCategories;
```

Comment:
Use base interface `List<Category>` instead of `ArrayList<Category>` - polymorphism in action